### ISSUE TITLE
Question about the number of frameskip of Franka Kitchen environment compare to LEXA.

### ISSUE THREAD

**Issue by Dongyeongkim** [seohongpark/METRA#3]
Hello,

First of all, thank you for your fabulous work in the field of unsupervised skill discovery.

I have a question about the frame_skip argument of Franka kitchen environment.

In LEXA, they use frame_skip as 16 
```python
self._env =  KitchenMicrowaveKettleLightTopLeftBurnerV0(frame_skip=16, control_mode = control_mode, imwidth=width, imheight=width)
```
[link](https://github.com/orybkin/lexa-benchmark/blob/674d56ea7454f7a3804a7ffb987930f64d6133e6/envs/kitchen.py#L17)

But I guess you use frame_skip as 40 https://github.com/seohongpark/METRA/blob/e08e76290ca7db7ecd0b4a0752383a17a7a536e4/lexa/lexa_envs/kitchen.py#L17

Do you have any reasons for that?

Thank you for reading.

Kind Regards,

Dongyeong Kim

**Comment by seohongpark** [comment]
Hi Dongyeong,

Thanks for the question.

40 comes from the default value defined at [this line](https://github.com/orybkin/lexa-benchmark/blob/674d56ea7454f7a3804a7ffb987930f64d6133e6/d4rl/kitchen/adept_envs/franka/kitchen_multitask_v0.py#L93). I remember I tested both 16 and 40 in some of my previous projects, found 40 to be better than 16 due to the reduced horizon at that time (though in a different setting), and used the same hyperparameter for METRA as well.

For our experiments in this work, we used the same `frame_skip` and maximum episode length (50) for all methods including LEXA to ensure a fair comparison.
